src/dev/flang/ast/ParsedCall.java

```diff
@@ -125,9 +125,30 @@ public ParsedCall(Expr target, ParsedName name, int select)
   }
 
 
+  public ParsedCall(ParsedCall thiz, AbstractFeature to, AbstractFeature from)
+  {
+    super(thiz, to, from);
+    _parsedName = thiz._parsedName;
+  }
+
+
   /*-----------------------------  methods  -----------------------------*/
 
 
+  /**
+   * When inheriting a post-condition during redefintion, this creates a clone
+   * of the inherited condition.
+   *
+   * @param to the redefining feature that inherits a contract
+   *
+   * @param from the redefined feature this contract should inherit from.
+   */
+  public Expr clonePostCondition(AbstractFeature to, AbstractFeature from)
```

Comment:
Not sure why this is needed? I would expect that ParsedCalls are not present anymore when cloning of post conditions happens?